cookbooks/aws-parallelcluster-platform/resources/install_packages/install_packages_rocky8.rb

```diff
@@ -20,11 +20,15 @@
 use 'partial/_install_packages_rhel_amazon.rb'
 
 action :install_kernel_source do
+  # Previous releases are moved into a vault area once a new minor release version is available for at least a week.
+  # https://wiki.rockylinux.org/rocky/repo/#notes-on-devel
   bash "Install kernel source" do
     user 'root'
     code <<-INSTALL_KERNEL_SOURCE
     set -e
-    dnf install -y #{kernel_source_package}-#{kernel_source_package_version} --releasever #{node['platform_version']}
+    kernel_source_rpm="#{kernel_source_package}-$(uname -r).rpm"
```

Comment:
Here we should use the `kernel_source_package_version` variable (which should already compatible with `uname -r`).